examples/plot_other_tracking.py

```diff
@@ -0,0 +1,129 @@
+        'full_segmented_cleaned_tractography.trk'),
+    op.join(
+        sub_path,
+        'sub-01_ses-01-dwi_tractography.trk'))
+
+afd.to_bids_description(
+    tractography_path,
+    **{"Name": "my_tractography",
+        "PipelineDescription": {"Name": "my_tractography"}})
+
+
+##########################################################################
+# Once this is done, you should have a folder in your home directory that
+# looks like this:
+#
+# | stanford_hardi
+# | ├── dataset_description.json
+# | └── derivatives
+# |     ├── freesurfer
+# |     │   ├── dataset_description.json
+# |     │   └── sub-01
+# |     │       └── ses-01
+# |     │           └── anat
+# |     │               ├── sub-01_ses-01_T1w.nii.gz
+# |     │               └── sub-01_ses-01_seg.nii.gz
+# |     ├── my_tractography
+# |     |   ├── dataset_description.json
+# |     │   └── sub-01
+# |     │       └── ses-01
+# |     │           └── dwi
+# |     │               └── sub-01_ses-01-dwi_tractography.trk
+# |     └── vistasoft
+# |         ├── dataset_description.json
+# |         └── sub-01
+# |             └── ses-01
+# |                 └── dwi
+# |                     ├── sub-01_ses-01_dwi.bvals
+# |                     ├── sub-01_ses-01_dwi.bvecs
+# |                     └── sub-01_ses-01_dwi.nii.gz
+
+
+##########################################################################
+# We specify the information we need to define the bundles that we will
+# segment. In this case, we are going to use a list of bundle names for the
+# bundle info. These names refer to bundles for which we already have
+# clear definitions of the information needed to segment them (e.g.,
+# waypoint ROIs and probability maps). For an example that includes
+# custom definition of bundle info, see the `plot_callosal_tract_profile`
+# example.
+
+bundle_info = ["SLF", "ARC", "CST", "FP"]
+
+##########################################################################
+# Now, we can run AFQ, pointing to the derivatives of the
+# "my_tractography" pipeline as inputs. This is done by setting the
+# `custom_tractography_bids_filters` key-word argument. We pass the
+# `bundle_info` defined above. We also point to the preprocessed
+# data that is in a `dmriprep` derivatives folder. These data were
+# preprocessed with 'vistasoft', so this is the pipeline we'll point to
+# If we were using 'qsiprep', this is where we would pass that
+# string instead. If we did that, AFQ would look for a derivatives
+# folder called 'stanford_hardi/derivatives/qsiprep' and find the
+# preprocessed DWI data within it. Finally, to speed things up
+# a bit, we also sub-sample the provided tractography. This is
+# done by defining the segmentation_params dictionary input.
+# To sub-sample to 10,000 streamlines, we define
+# `'nb_streamlines' = 10000`.
+
+my_afq = api.AFQ(
+    bids_path,
+    dmriprep='vistasoft',
+    bundle_info=bundle_info,
+    custom_tractography_bids_filters={
```

Comment:
May want to include primer on how to configure/diagnose bids_filters:

```
bids_layout = BIDSLayout(BIDS_DIR, derivatives=True)
# find all files
bids_layout.files
# get a specific file; probably want to specify fill path
bids_layout.get()
# find out what entities are available for bids filters for file
bids_layout.get().get_entities()
```

Some issues have been `dataset_description.json`, file naming convention doesn't uniquely identify file with bids filters, ...